Description: drop one test that fails now
Author: Yadd <yadd@debian.org>
Bug-Debian: https://bugs.debian.org/1002752
Forwarded: no
Last-Update: 2021-12-29

--- a/test/fixtures/valid/jsx1.jsx
+++ /dev/null
@@ -1,29 +0,0 @@
-<Global.Nav.Component {...props}
-    closeButton=<CloseButton / >
-    // Temporary logo
-    logo={<img src="https://example.com/tmp.png" />}
-    /* logo={<img src="https://example.com/logo.png" />} */>
-  <//c1
-    /*c2*/  xml:nav-item//c2
-  /*c3*/>
-    line one
-    next line
-  <//c4
-    /*c5*///*c6*/
-    /*c7*// /*c8*/
-    /*c9*/xml:nav-item//c10
-  /*c11*/>
-
-  <div>
-    <p>
-      {/* comment */}
-      <a href={`https://${domain}.com/${<path>{`${{}<x>y</x/g}`}</path>.toString()}`}
-        alt/**/=/**/<Alt text=<Text/>/>
-      >
-        Some{' '}link
-      </a>
-      {// comment
-      }
-    </p>
-  </div>
-</Global.Nav.Component>;
